website/docs/components/link/standalone/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,114 @@
+## Usage
+
+### When to use
+
+- To navigate to a new destination, either internally or externally.
+- As a standalone element or within a Button Set, use a **(Link) Standalone**
+
+### When not to use
+
+- Within a body of text, use a **Link (Inline)**
+- To trigger an action or event, like a form submission, use a [Button](/components/button/overview)
+
+### Size
+
+<Hds::Link::Standalone @size="large" @text="Large" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+<Hds::Link::Standalone @size="medium" @text="Medium" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+<Hds::Link::Standalone @size="small" @text="Small" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+
+Medium is the preferred size, but use a link size that best fits the UI.
+
+For example:
+
+- Don't use large links in tables.
+
+### Type
+
+#### Primary
+
+Use the `primary` variant as the default and for more important links. 
+
+For example when navigating the user to `View clusters`:
+
+<Hds::Link::Standalone @text="View clusters" @icon="arrow-right" @iconPosition="trailing" @type="primary" @route="components" />
+
+!!! Info
+
+Take care when needing multiple links on a page; using too many primary links can detract from the end goal.
+!!!
+#### Secondary
+
+Use the Secondary variant for less important links, when the action color can't be used, or when creating a list of links.
+
+For example in an Install page, 'Learn more about Vault'
+
+<Hds::Link::Standalone @text="View clusters" @icon="external-link" @iconPosition="trailing" @color="secondary" @route="components" />
+
+### State
+#### Primary
+
+TBD Image here?
```

Comment:
@jorytindall  this is an example where we have this image in the existing guidelines but i'm not sure it is possible to recreate in code

Reply:
I think either way I should move these to the Specifications tab

Reply:
moved these to the correct place and added images